vendor/engines/sanger_sequencing/app/controllers/sanger_sequencing/base_controller.rb

```diff
@@ -0,0 +1,15 @@
+module SangerSequencing
+
+  class BaseController < ApplicationController
+
+    before_action :assert_sanger_enabled_for_facility
+
+    private
+
+    def assert_sanger_enabled_for_facility
+      raise ActionController::RoutingError, "Sanger not enabled for this facility" unless current_facility.sanger_sequencing_enabled?
```

Comment:
`current_facility` might be `nil`. Implementing some kind of a `NullFacility` object is on my wish list.